lib/Ix/App.pm

```diff
@@ -60,7 +62,22 @@ sub _build_psgi_app ($self) {
       ];
     }
 
-    my $ctx = $self->processor->context_from_plack_request($req);
+    my ($ctx, $exception);
+
+    try {
```

Comment:
Discussed on Slack.  This should be moved up into App's general "big eval wrapper" so the "handle HTTP::Exceptions?" comment becomes code.